pulp_smash/constants.py

```diff
@@ -12,6 +12,13 @@
     http://pulp.readthedocs.io/en/latest/dev-guide/conventions/sync-v-async.html#call-report
 """
 
+CONTENT_SOURCE_ID = 'content-world'
```

Comment:
We should be creating random IDs using uuid calls. This id was an example on documentation.